Userland/Libraries/LibGL/GLContext.cpp

```diff
@@ -1945,6 +1945,16 @@ void GLContext::gl_depth_mask(GLboolean flag)
     m_rasterizer.set_options(options);
 }
 
+void GLContext::gl_clip_plane(GLenum plane, [[maybe_unused]] GLdouble const* equation)
+{
+    APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED(gl_clip_plane, plane, equation);
+
+    RETURN_WITH_ERROR_IF(m_in_draw_state, GL_INVALID_OPERATION);
+    RETURN_WITH_ERROR_IF((plane < GL_CLIP_PLANE0) || (plane > GL_CLIP_PLANE5), GL_INVALID_ENUM);
+
+    // This is a stub!
```

Comment:
You can replace this with `dbgln_if(GL_DEBUG, ...)` to make it stand out for our future debugging selves ;-)